examples/remotevlan_ovs/README.md

```diff
@@ -1,15 +1,16 @@
 # NSM Remote Vlan Examples
 
-This setup can be used to check remote vlan mechanism with both OVS and VPP forwarder. Contain basic setup for NSM that includes `nsmgr`, `registry-k8s`, `admission-webhook-k8s`, `nse-remote-vlan`. The `nse-remote-vlan` belongs to the nsm-system since does not have role in data-plane connection.
+This setup can be used to check remote vlan mechanism with OVS orwarder. Contain basic setup for NSM that includes `nsmgr`, `registry-k8s`, `forwader-ovs`, `admission-webhook-k8s`, `nse-remote-vlan`. The `nse-remote-vlan` belongs to the nsm-system since does not have role in data-plane connection.
```

Comment:
```suggestion
This setup can be used to check remote vlan mechanism with OVS forwarder. Contain basic setup for NSM that includes `nsmgr`, `registry-k8s`, `forwader-ovs`, `admission-webhook-k8s`, `nse-remote-vlan`. The `nse-remote-vlan` belongs to the nsm-system since does not have role in data-plane connection.
```